src/sentry/api/event_search.py

```diff
@@ -584,6 +585,10 @@ def convert_search_filter_to_snuba_query(search_filter):
                 )
             )
         return [name, search_filter.operator, internal_value]
+    elif name == "trace":
+        if search_filter.value.raw_value:
```

Comment:
We can assume that `SearchValue()` is always applied to the `value` field for the `SearchFilter` namedtuple; thus,  we can assume that `search_filter.value.raw_value` always exists.

Reply:
The value could be blank though in the case of `trace: foo:bar`

Reply:
Oh interesting.